packtools/sps/validation/article_citations.py

```diff
@@ -10,7 +10,9 @@ def __init__(self, xmltree, citation, publication_type_list=None):
         self.citation = citation
         self.publication_type_list = publication_type_list
 
-    def validate_article_citation_year(self, start_year=None, end_year=None):
+    def validate_article_citation_year(
```

Comment:
@Rossi-Luciano nesta classe crie `def validate_citation_journal_volume_or_number(` quando o tipo da citação for `journal`.